.rubocop.yml

```diff
@@ -149,6 +149,9 @@ MethodLength:
 ModuleFunction:
   Enabled: false
 
+MutableConstant:
+  Enabled: false
```

Comment:
Ah right, I forgot that without `.freeze` this will raise an error. Thanks for making this change.